core/kazoo_json/test/kz_json_test.erl

```diff
@@ -557,3 +557,21 @@ sum_test_() ->
     ,?_assertEqual(A42Bhi, kz_json:sum(A2Bhi, A40))
     ,?_assertEqual(?CHARGES_DOUBLE, kz_json:sum(?CHARGES_SIMPLE, ?CHARGES_SIMPLE))
     ].
+
+from_list_recursive_test() ->
+    List = [{<<"fax">>,
+             [{<<"id">>,<<"201702-1b6cfec4e3ab0b7bb97ced78bf431f39">>},
+              {<<"info">>,
+               [{<<"fax_hangup_code">>,200},
+                {<<"fax_hangup_cause">>,<<"NORMAL_CLEARING">>}]},
+              {<<"notifications">>,
+               {[{<<"email">>,
+                  {[{<<"send_to">>,
```

Comment:
This and above are literal jobjs. I thought the idea was that this should be a proplist that would get transformed into jobjs by your function?